app/views/job_runs/discovery_report_summary.html.erb

```diff
@@ -1,3 +1,7 @@
+
+<%# In current reports, objects_with_error is an array; in legacy, it was an integer. %>
+<% objects_with_error = @discovery_report['summary']['objects_with_error'] %>
+<% objects_with_error_size = objects_with_error.is_a?(Array) ? objects_with_error.size : objects_with_error %>            <td class="text-center"><%= @discovery_report['summary']['objects_with_error'].size %></td>
```

Comment:
Does the `<td>` at the end of this line belong there or elsewhere?

Reply:
Or maybe it's a mis-paste and can just go away given the change on line 29

Reply:
That's where it went ... fixing.